src/gradualizer_fmt.erl

```diff
@@ -342,6 +346,20 @@ format_type_error({bad_type_annotation, TypeLit}, Opts) ->
       [format_location(TypeLit, brief, Opts),
        pp_expr(TypeLit, Opts),
        format_location(TypeLit, verbose, Opts)]);
+format_type_error({internal_error, form_check_timeout, Form}, Opts) ->
+    io_lib:format(
+      "~sTimeout checking ~s~n",
+      [format_location(Form, brief, Opts),
+       case proplists:get_value(fmt_location, Opts, ?FMT_LOCATION_DEFAULT) of
+           brief ->
+               "form";
+           verbose ->
+               io_lib:format("~s~s~n"
+                             "This is most likely a bug in Gradualizer.~n"
+                             "Please report it at https://github.com/josefs/Gradualizer/issues",
+                             [form_info(Form),
+                              format_location(Form, verbose, Opts)])
```

Comment:
The `fmt_location` isn't meant to be a generic `--verbose` flag. It was added because someone was afraid of just changing the location output to the [GNU standard](https://www.gnu.org/prep/standards/html_node/Errors.html) file:line:col.

I think it should be possible to use brief location and still be use a verbose message. (Furthermore, I think "on line x column y" should be deleted, but that's a different story.)

Reply:
I think the `brief` format plays better with tools which want to integrate with Gradualizer, both due to the easier to parse location format and due to the shorter message. I imagine these could have independent switches in the future, but for the time being I'd try to keep the message brief too, if we ask for the brief location format.

That being said, we shouldn't completely omit a vital part of the message, so I've extended the brief text. Before:
![Screenshot 2022-02-22 at 18 05 46](https://user-images.githubusercontent.com/112145/155182499-4d86ab50-5654-46af-8b02-039f1892be10.png)

After:
![Screenshot 2022-02-22 at 18 14 28](https://user-images.githubusercontent.com/112145/155183737-d072cc55-93f2-433f-97ba-06051115b3b7.png)